content/docs/0.8.x/monitoring/prometheus/install/index.md

```diff
@@ -25,51 +41,54 @@ Some environment variables have to set up in the prometheus-service deployment
 ```yaml
```

Comment:
Can you please move this section (starting with: *Some environment variables have to set up in the prometheus-service deployment*) to the bullet point: `* Replace the environment variable value according to the use case and apply the manifest`

Reply:
DOne